kinto/core/resource/__init__.py

```diff
@@ -918,9 +887,8 @@ def _raise_400_if_id_mismatch(self, new_id, record_id):
     def _extract_partial_fields(self):
         """Extract the fields to do the projection from QueryString parameters.
         """
-        fields = self.request.GET.get('_fields', None)
+        fields = self.request.validated.get('querystring', {}).get('_fields')
```

Comment:
Ditto: could this happen that `querystring` is missing? If it's for the test, we could fix the test setUp instead I believe.